app/static/js/userProfile.js

```diff
@@ -140,9 +141,24 @@ function displayMessage(message, color) {  // displays message for saving backgr
     setTimeout(function() {$("#displaySave").html("").removeClass("text-"+ color)}, 2000)
 }
 
-function updateManagers(el, volunteer_username ){// retrieve the data of the studnet staff and program id if the boxes are checked or not
-  var program_id=$(el).attr('data-programid');
-  action= el.checked ? 'add' : 'remove';
+$(function() {
+     toastElementList = [].slice.call(document.querySelectorAll('.toast'))
+     toastList = toastElementList.map(function (toastEl) {
+        return new bootstrap.Toast(toastEl)
```

Comment:
Is there a name that would work better that also describes what the list contains within our application?

Reply:
Would it better to have a generic name for the element list if they want to be used in the future for something else? I can rename it for sure, just a thought.